sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -17,13 +17,28 @@
 
 package org.apache.spark.sql.execution.datasources.jdbc
 
+import java.sql.Connection
+import java.util.Properties
+
 /**
  * Options for the JDBC data source.
  */
 class JDBCOptions(
     @transient private val parameters: Map[String, String])
   extends Serializable {
 
+  import JDBCOptions._
+
+  def this(url: String, table: String, parameters: Map[String, String]) = {
+    this(parameters ++ Map("url" -> url, "dbtable" -> table))
+  }
+
+  val asProperties: Properties = {
+    val properties = new Properties()
+    parameters.foreach { case (k, v) => properties.setProperty(k, v) }
```

Comment:
We should exclude `url` and `dbtable` here. Otherwise, we will pass them to the `connect` call of JDBC driver.